docker-compose-generator/docker-fragments/opt-add-bluewallet-lndhub.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  bluewallet_lndhub_redis:
+    container_name: bluewallet_lndhub_redis
+    image: "redis:6.2.2-buster@sha256:e10f55f92478715698a2cef97c2bbdc48df2a05081edd884938903aa60df6396"
+    command: "redis-server --requirepass S7bk49NBp29khUci4Zw28vsBviCnYfZ2QLTEsQNizq9ni5ur"
+    restart: "on-failure"
+    stop_grace_period: "1m"
+    init: true
+    volumes:
+      - "bluewallet_lndhub_redisdir:/data"
+
+
+  bluewallet_lndhub_app:
+    container_name: bluewallet_lndhub_app
+    image: "bluewalletorganization/lndhub:v1.3.3@sha256:b99408dfb53f503def405668f5fc4d8185de39a87843d4ea8774544a5df1fe36"
+    user: "0:0"  #need help here as the official docker image from bluewalletorganization uses a user called lndhub, which by default cant read macaroon from BTCPay Server LND.  How to get around this without giving root access?
```

Comment:
I think it's ok to do it like that. I had to resort to the same solution in the Sphinx Relay integration, see [here](https://github.com/btcpayserver/btcpayserver-docker/pull/458/files#diff-2e6995bab7ee08c2021bca3ea8640707c8adaa667e8699619d32b1f4923721e1R13)

Reply:
@dennisreimann Thanks a lot for your review!  Cool, I could not think of any other way to do that, without building a new image myself.

Reply:
Yep, I also researched this and came to the same conclusion. Having a separate image would be tedious to maintain.